release-notes/release-notes59.md

```diff
@@ -219,27 +219,27 @@ The {{site.productname}} 5.9 release includes an accompanying release of the **P
 
 For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/).
 
-### Spell Checker Pro 2.4.0
+### Spell Checker Pro 2.4.1
 
 The {{site.productname}} 5.9 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
 
-**Spell Checker Pro** 2.4.0 adds the following new features:
+**Spell Checker Pro** 2.4.1 adds the following new features:
```

Comment:
Could this get confusing since it's a patch release and has new features?

Reply:
```suggestion
**Spell Checker Pro** 2.4 (2.4.1) adds the following new features:
```
?